warehouse/templates/pages/help.html

```diff
@@ -183,6 +185,16 @@ <h3 id="login-problem">{{ login_problem() }}</h3>
         </p>
         {{ code_of_conduct() }}
 
+        <h3 id="tls-deprecation">{{ tls_deprecation() }}</h3>
+        <p>
+          Transport Layer Security, or TLS, is part of how we make sure connections between your computer and PyPI are private and secure. It's a cryptographic protocol that's had several versions over time. PyPI <a href="https://mail.python.org/pipermail/python-announce-list/2018-April/011885.html" target="_blank" rel="noopener">turned off support for TLS versions 1.0 and 1.1</a> in April 2018 (<a href="https://pyfound.blogspot.com/2017/01/time-to-upgrade-your-python-tls-v12.html" target="_blank" rel="noopener">reason</a>). If you are having trouble with <tt> pip install</tt> and get a <tt>No matching distribution found</tt> or <tt>Could not fetch URL</tt> error, try adding <tt>-v</tt> to the command to get more information:
+        </p>
+        <p><tt>pip install --upgrade -v pip</tt>
+        </p>
+        <p>If you see an error like <tt>There was a problem confirming the ssl certificate</tt> or <tt>tlsv1 alert protocol version</tt> or <tt>TLSV1_ALERT_PROTOCOL_VERSION</tt>, you need to be connecting to PyPI with a newer TLS support library. The specific steps you need to take will depend on your operating system version, where your installation of Python originated (python.org, your OS vendor, or an intermediate distributor), and the installed versions of Python and <tt>pip</tt>. For help, please go to <a href="https://webchat.freenode.net/?channels=%23pypa" target="_blank" rel="noopener">the <tt>#pypa</tt> IRC channel on Freenode</a>, file an issue at <a href="https://github.com/pypa/packaging-problems/issues" target="_blank" rel="noopener">pypa/packaging-problems/issues</a>, or <a href="https://www.python.org/community/lists/" target="_blank" rel="noopener">post to the python-help mailing list</a>, including your OS and installation details and the output of <tt>pip install -vvv upgrade pip</tt>.
```

Comment:
Are we sure the "There was a problem confirming the ssl certificate" error is actually this? I wouldn't expect to see this error message.

Reply:
Some users do get "There was a problem confirming the ssl certificate" as part of the error for this issue -- I looked through https://github.com/issues?utf8=%E2%9C%93&q=%22there+was+a+problem+confirming+the+ssl+certificate%22+user%3Apypa+ and verified.

Reply:
"There was a problem confirming the ssl certificate: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version" gosh that's an awful error message :-(

Okey doke! Thanks for putting this together.